docs/src/main/asciidoc/security-built-in-authentication-support-concept.adoc

```diff
@@ -10,11 +10,11 @@ include::_attributes.adoc[]
 
 The following section describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC, and Mutual TLS authentication. Proactive authentication is also described.
 
-include::security-basic-authentication-concept.adoc[leveloffset=+1]
+== Basic Authentication
```

Comment:
@rolfedh - I'm just wondering whether this section belongs in this adoc or in `security-basic-authentication-concept.adoc`?